demo/simple-browser.lisp

```diff
@@ -121,11 +121,70 @@ Loads and renders a single web page."
       (webkit2:webkit-web-view-load-uri view "hello:stranger")
       (gtk:gtk-widget-show-all win))))
 
-(defun test-browser-main (&key private extended styled custom-scheme)
+(defun js-transform-browser-main ()
+  "A JS transformation-checking version of `simple-browser-main'."
```

Comment:
Maybe "validating" instead of "checking" and "variation" / "variant" instead of "version".